### PR TITLE
[SGD] Support multiple input model

### PR DESCRIPTION
**Pull request by ConeyLiu**
## Why are these changes needed?

Add support for multiple input model like the follows:
```python
def forward(self, x, y, z):
   pass
```

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/latest/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failure rates at https://ray-travis-tracker.herokuapp.com/.
- Testing Strategy
   - [x] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested (please justify below)

### CODE REVIEW

**Comment by ConeyLiu** [comment]
Hi @richardliaw, could you help to review this? Thanks a lot.

File: python/ray/util/sgd/torch/training_operator.py [hunk 1]
```diff
@@ -237,15 +237,18 @@ def train_batch(self, batch, batch_info):
                 calculate averages.
 
         """
-        features, target = batch
+        # unpack features into list to support multiple inputs model
+        *features, target = batch
```
Comment (richardliaw) [inline python/ray/util/sgd/torch/training_operator.py:241]: Looks good. can you update the docstring?

**Review by richardliaw — APPROVED** [review]

File: python/ray/util/sgd/torch/training_operator.py (hunk 1 above)
Comment (ConeyLiu) [reply python/ray/util/sgd/torch/training_operator.py:242]: updated

**Comment by ConeyLiu** [comment]
thanks @richardliaw .

**Pull request merged**